rando.py

```diff
@@ -89,8 +91,34 @@ def cointoss(self, mess, args):
 
     @botcmd(admin_only=False)
     def diceroll(self, mess, args):
-        """Returns a dice number (1 through 6)"""
-        return random.randint(1, 6)
+        """Returns a 6-sided dice roll. Optional argument like '3w6 1w12'."""
```

Comment:
I'm curious about this `...w...` syntax. What does it mean exactly? Is it something that's used in dice games or something?

Reply:
Thanks @coyotwill for the insight! In light of that, I think we should use "d" everywhere (docstring, help message, etc.), but I'm ok with keeping `wW` in the regex for your convenience.